gui/src/main.ts

```diff
@@ -87,6 +88,7 @@ Vue.component("cortx-has-access", CortxHasAccess);
 Vue.component("cortx-info-tooltip", CortxInfoTooltip);
 Vue.component("cortx-confirmation-dialog", CortxConfirmationDialog);
 Vue.component("cortx-dropdown", CortxDropdown);
+Vue.component("cortx-reset-password", CortxResetPasswrod);

```

Comment:
We don't have to register it globally. The components which need it, will import it.

Reply:
Need to use this component at the place of account change password as well. Hence we can register it globally.